server/pxf-service/src/test/java/org/greenplum/pxf/service/BridgeOutputBuilderTest.java

```diff
@@ -129,6 +129,7 @@ public void testCSVSerialization() throws Exception {
         addColumn(context, 12, DataType.DATE, "col12");
         addColumn(context, 13, DataType.VARCHAR, "col13");
         addColumn(context, 14, DataType.VARCHAR, "col14");
+        addColumn(context, 14, DataType.TEXT, "col15");
```

Comment:
```suggestion
        addColumn(context, 15, DataType.TEXT, "col15");
```
wonder why nothing failed because of the wrong index here ?